app/routes/posts/show.js

```diff
@@ -12,6 +17,20 @@ export default Ember.Route.extend({
       model.save().then(function(data) {
         route.transitionTo('posts');
       });
-    }
-  }
+    },
+    createNewComment: function(body) {
+      let post = this.controller.get('model');
+      let user = this.store.findRecord("user", 1);
+      const commentCreation = this.store.createRecord('comment', {
+        body: body,
+        post: post,
+        user: user
+      });
+      commentCreation.save();
+      $(".ember-text-area").val("").focus();
```

Comment:
This is to empty the text area after a comment has been saved. Please let me know if I should look into ember specific ways of emptying a text area.